### DISCUSSION TITLE
yiwen-blog-website/blog/aws_use

### DISCUSSION THREAD

**Discussion by giscus** [Hao-yiwen/yiwen-blog-website#109] [Announcements]
# yiwen-blog-website/blog/aws_use

今天因为需要购买了一台ec2服务器，但是在登录实例的时候发现始终无法登录，后面对登录进行了研究，ec2不同于国内的ecs等服务器，ec2的登录只能使用密钥对登录，而如果在创建实例的时候没有设置密钥对，那么则需要自己在服务器中设置，而不是控制面板。以下是详细流程:

https://hao-yiwen.github.io/yiwen-blog-website/blog/aws_use

**Comment by Hao-yiwen** [comment]
使用ec2做代理中转同样有巨大的被墙风险。
